iopipe/context.py

```diff
@@ -34,6 +42,23 @@ def log(self, key, value):
 
         self.instance.report.custom_metrics.append(event)
 
+    def log(self, key, value):
+        self.metric(key, value)
+
+    def label(self, name):
+        if not isinstance(name, str):
```

Comment:
This check will be problematic as in Python 2.7 `unicode` is also a string type, but does not exist in Python 3.x. You can borrow the string types handling in the popular `six` library to make this Python 2/3 compatible.

Reply:
should I do `type(name) == 'str'`?  that means we wouldn't have to add the six library to deps